Lesson34/selcan/lesson34-app/src/components/Form/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from "react";
+import { Account } from "../Account";
+
+// TODO: Create a new Component that contains log in form, so that user can type username and password and click submit button to log in.
+// If user is logged in, show Account component.
+// If user is logged out, show Log In component.
+
+export default function Login() {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+  const [loggedIn, setLoggedIn] = useState("");
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    if (username === "user" && password === "password") {
+      setLoggedIn(true);
+    } else {
+      alert("wrong username or password!");
+    }
+  };
+
+  return (
+    <div>
+      {loggedIn ? (
+        <Account />
```

Comment:
Remove this condition. It's better to only keep login form related stuff in this component. In the App.js component, you can then decide what to show on the screen: login form or account information (move this check to the App.js file).